main/tests/server/src/com/google/refine/grel/GrelTests.java

```diff
@@ -226,6 +230,37 @@ public void testCrossFunctionEval() throws ParsingException {
         Assert.assertTrue(result instanceof EvalError);
     }
 
+    @Test
+    public void testColumnDependencies() throws ParsingException {
+        // integration test for column dependency extraction
+
+        String baseColumn = "base";
+        String tests[][] = {
+                { "value", "base" },
+                { "cell.recon.match.id", "base" },
+                { "value + 'a'", "base" },
+                { "\"constant\"", "" },
+                { "1", "" },
+                { "cells.foo", "foo" },
+                { "value + ' ' + cells.foo.value", "base,foo" },
+                { "cells[\"foo\"].value+'_'+value", "base,foo" },
+                { "parseHtml(value.trim())", "base" },
+                { "cells", null },
+                // this could be analyzed too, but we will never reach completeness anyway!
+                { "get(cells, 'foo'+'bar')", null },
```

Comment:
I don't think I understand the comment here. Isn't the column dependency whatever the second argument evaluates to? ie "foobar" in this case.